align/cell_fabric/canvas.py

```diff
@@ -143,6 +143,140 @@ def join_wires(self, wire, exclude_nets=None, include_nets=None, max_length=None
                         new_length = 0
         self.terminals = self.removeDuplicates(allow_opens=True).copy()
 
+    def drop_via(self, via, exclude_nets=None, include_nets=None):
```

Comment:
You should look at the coverage you have with your tests (https://33498-177476440-gh.circle-artifacts.com/0/coverage-reports/align/align_cell_fabric_canvas_py.html).
![image](https://user-images.githubusercontent.com/1512574/110549311-ba380900-80e6-11eb-8d3c-45e570aea6c7.png)

You might be able to improve it by testing both horizontal and vertical cases, etc.

Reply:
Great BKM. I will increase coverage in a new PR.

Reply:
Is it possible to generate a similar report while running pytest locally?